<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>316</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="empty"/>
      <widget class="QWidget" name="connection_edit">
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="not_connected_message">
          <property name="text">
           <string>Notification</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Enter new address:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_host">
          <property name="text">
           <string>Host:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="edit_host">
          <property name="text">
           <string>debian</string>
          </property>
          <property name="placeholderText">
           <string>host</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_port">
          <property name="text">
           <string>Port:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="edit_port">
          <property name="inputMethodHints">
           <set>Qt::ImhPreferNumbers</set>
          </property>
          <property name="text">
           <string>9090</string>
          </property>
          <property name="placeholderText">
           <string>port</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="update_address">
          <property name="text">
           <string>Apply new Configuration</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="setting_name_file">
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>140</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Choose name and file</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="setting_name_edit">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>61</y>
          <width>559</width>
          <height>33</height>
         </rect>
        </property>
        <property name="text">
         <string>basement</string>
        </property>
        <property name="placeholderText">
         <string>setting</string>
        </property>
       </widget>
       <widget class="QPushButton" name="choose_image">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>100</y>
          <width>141</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Select Image</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>559</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>connection established</string>
        </property>
       </widget>
       <widget class="QGraphicsView" name="graphicsView">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>140</y>
          <width>251</width>
          <height>201</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>131</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Preview of image:</string>
        </property>
       </widget>
       <widget class="QPushButton" name="set_setting">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>200</y>
          <width>96</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Set</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="add_sensors">
       <widget class="QTableWidget" name="tableWidget">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>20</y>
          <width>256</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="select_setting">
       <layout class="QVBoxLayout" name="verticalLayout_2"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>595</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
